webviz_subsurface/_datainput/eclipse_init_io/pvt_water.py

```diff
@@ -298,6 +320,38 @@ def viscosity_unit(self, latex: bool = False) -> str:
             return fr"${unit_system.viscosity().symbol}$"
         return f"{unit_system.viscosity().symbol}"
 
+    def density_unit(self, latex: bool = False) -> str:
+        """Creates and returns a string containing the unit symbol of the density.
+
+        Args:
+            latex: True if the unit symbol shall be returned as LaTeX, False if not.
+
+        Returns:
+            A string containing the unit symbol of the density.
+
+        """
+        unit_system = EclUnits.create_unit_system(
+            self.original_unit_system
+            if self.keep_unit_system
+            else EclUnitEnum.ECL_SI_UNITS
+        )
+
+        if latex:
+            return fr"${unit_system.density().symbol}$"
+        return f"{unit_system.density().symbol}"
```

Comment:
Is `unit_system.density().symbol` a string? If so we can do:
```suggestion
        return unit_system.density().symbol
```